specifications/passive/android_phone-1.0.0.yml

```diff
@@ -134,7 +134,20 @@ data:
     sample_rate:
       interval: 3600
       configurable: true
-  # Usage
+
+
+  -type: PHONE_BLUETOOTH_DEVICES_SCANNED
+     app_provider: .phone.PhoneBluetoothService
+     unit: NON_DIMENSIONAL
+     processing_state: RAW
+     topic: android_phone_bluetooth_devices_scanned
+     value_schema: .passive.phone.PhoneBluetoothDevicesScanned
+     sample_rate:
+      interval: 3600
+      configurable: true
```

Comment:
Please use same indentation as rest of the file and remove an whitespace before and after this block.

Also sample rate probably is not configurable for this one but instead depends on the sampling rate of PHONE_BLUETOOTH_DEVICES so you can remove the configurable key.

Reply:
Ya! Will do...

Reply:
Ya! I will do it...